src/junit.rs

```diff
@@ -167,13 +148,13 @@ pub fn use_reader(
                 b"skipped" => {
                     let testrun = saved_testrun
                         .as_mut()
-                        .ok_or_else(|| ParserError::new_err("Error accessing saved testrun"))?;
+                        .ok_or_else(|| anyhow!("Error accessing saved testrun"))?;
```

Comment:
The `.context()` method also exists on `Option`, so all these can be further simplified :-)